index.js

```diff
@@ -1,21 +1,38 @@
 'use strict';
 
-module.exports = {
-  env: {
-    node: true,
-  },
-  extends: [
-    'crowdstrike',
-    'plugin:n/recommended',
-  ],
-  plugins: [
-    'json-files',
-  ],
-  rules: {
-    'strict': 'error',
+const jsonFiles = require('eslint-plugin-json-files');
+const globals = require('globals');
+const { FlatCompat } = require('@eslint/eslintrc');
+const js = require('@eslint/js');
+const nodePlugin = require('eslint-plugin-n');
 
-    'json-files/require-engines': 'error',
-    'json-files/require-license': 'error',
-    'json-files/restrict-ranges': ['error', { pinUnstable: true }],
-  },
-};
+const compat = new FlatCompat({
+  baseDirectory: __dirname, // optional; default: process.cwd()
+  resolvePluginsRelativeTo: __dirname, // optional
+  recommendedConfig: js.configs.recommended, // optional unless using "eslint:recommended"
+});
+
+module.exports = [
+  ...compat.extends('crowdstrike'),
+  nodePlugin.configs['flat/recommended-script'],
+  {
+    plugins: {
+      'json-files': jsonFiles,
+    },
+    processor: "json-files/json",
```

Comment:
Not sure what's going on here. This should fail linting with https://github.com/CrowdStrike/eslint-config-crowdstrike/blob/main/index.js#L28?

Reply:
Fixed it by specifying those rules/processor only for `*.json` files